pages/pipelines/command_step.md.erb

```diff
@@ -120,12 +120,51 @@ _Optional attributes:_
       <em>Example:</em> <code>"My reason"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>retry</code></td>
+    <td>
+      Configuration of how retries (either <strong>automatic</strong> or <strong>manual</strong>) operate on this step<br>
+      Note: If automatic retries are enabled, a retry job will be triggered if it fails with any exit status <code>"*"</code> a maximum of 2 times.<br>
+
+      <br/>
+      <em>Example (automatic retries with defaults):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic: true</code><br/>
+
+      <br/>
+      <em>Example (different automatic retry rules depending on exit status):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;automatic:</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: 5</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;limit: 2</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;- exit_status: "*"</code><br/>
+      <code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;limit: 4</code><br/>
+
+      <br/>
+      <em>Example (disabling manual retries):</em><br>
+      <code>retry:</code><br>
+      <code>&nbsp;&nbsp;manual: false</code><br>
```

Comment:
@keithpitt does `manual` default to true? And can a "manual" retry come from the API as well as the app, or only from the button in the app?

Reply:
That's right, `manual: true` is the default. A "manual" retry is a retry initiated by a user, either via the API or UI. There's no difference to BK between an API retry or a UI retry.